views/account/org/dashboard.jade

```diff
@@ -1,10 +1,26 @@
 extends ../../app_layout
 
 block content
-  .col.s12
-    h2 Organization Dashboard
-    .col-sm-12
-      a.btn.btn-primary(href="/current-survey") Current Survey
-      a.btn.btn-primary(href="/results") Results
-    .col-sm-12
-      a.btn.btn-primary(href="/employee/dashboard") Employee Dashboard
+
+  .container
+    .row
+      h2(align="center") Organization Dashboard
+  .col.s10.offset-s1.m8.offset-m2.l6.offset-l3
```

Comment:
This doesn't seem to be indented correctly, line 8 should be inside .container, no?